packages/js/src/generators/library/library.ts

```diff
@@ -240,12 +251,26 @@ function addProject(tree: Tree, options: NormalizedSchema) {
     }
 
     if (options.publishable) {
-      const publishScriptPath = addMinimalPublishScript(tree);
+      const packageRoot = join(defaultOutputDirectory, '{projectRoot}');
 
-      projectConfiguration.targets.publish = {
-        command: `node ${publishScriptPath} ${options.name} {args.ver} {args.tag}`,
-        dependsOn: ['build'],
+      projectConfiguration.targets['nx-release-publish'] = {
```

Comment:
nit: its not **super** important yet, but for future it may be better if we go ahead and do something like this so that when all the targets are inferred we don't have issues with the targets block being undefined.

```suggestion
      projectConfiguration.targets ??= {}
      projectConfiguration.targets['nx-release-publish'] = {
```

Reply:
updated 👍